extensions/asset/README.md

```diff
@@ -0,0 +1,29 @@
+# Asset Definition Specification (`asset`)
+
+**Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+A Collection extension to provide details about assets that are available in member Items.
+
+- [Example](examples/example-landsat8.json)
+- [JSON Schema](json-schema/schema.json)
+
+## Collection fields
+
+This extension introduces a single new field, `assets` at the top level of a collection. An Asset Object defined at the Collection level is nearly the same as the [Asset Object in Items]((../../item-spec/item-spec.md#asset-object)), except for two differences. The `href` field is not required, because collections don't point to any data by themselves. Additioanlly the remaining fields, `title` and `type` are required in the Asset Definition, in order for it to adequately describe Item assets.
+
+| Field Name         | Type   | Description                                                  |
+| ------------------ | ------ | ------------------------------------------------------------ |
+| assets | Map<string, Asset Object | **REQUIRED.** A dictionary of assets that can be found in member Items |
+
+### Asset Object
+
+An asset is an object that contains details about the datafiles that will be included in member Items. Assets included at the Collection level do not imply that all assets are available from all Items. However, it is recommended that the Asset Definition is a complete set of all assets that may be available from any member Items.
```

Comment:
I think it should say that the table below is not a complete list of members and can potentially include other properties, especially from extensions (bands from eo and sar).